src/lua/init.lua

```diff
@@ -206,8 +206,39 @@ rawset(package, "setsearchroot", setsearchroot)
 local compat = require('compat')
 package.loaded.compat = nil
 
+-- Execute scripts or load modules pointed by TT_PRELOAD
+-- environment variable.
+local function run_preload()
+    local tt_preload = os.getenv('TT_PRELOAD') or ''
+    assert(type(tt_preload) == 'string')
```

Comment:
Minor: I see no reason for this `assert`. Does it try to prevent the case when `os.getenv` result value type is changed?

Reply:
Nope, it actually has no sense. Removed.